app/konnect/runtime-manager/kong-gateway-runtime.md

```diff
@@ -59,11 +58,186 @@ representative for access.
     <!-- and the tag in the **Node Status** column should say
     **Connected**.-->
 
+{% endnavtab %}
+{% navtab Advanced %}
+
+1. From the left navigation menu, open **Runtimes**.
+
+    For the first runtime, the page opens to a **Configure New Runtime** form.
+
+    Once configured, this page lists all runtimes associated with the
+    {{site.konnect_short_name}} SaaS account.
+
+2. (Optional) If this is not the first runtime configuration, click
+**Configure New Runtime**.
+
+3. Click **Generate Certificate**.
+
+    Three new fields appear: a certificate, a private key, and a root CA
+    certificate. The contents of these fields are unique to each
+    runtime configuration.
+
+4. Save the contents of each field into a separate file in a safe location:
+
+    * Certificate: `cluster.crt`
+    * Private key: `cluster.key`
+    * Root CA Certificate: `ca.crt`
+
+    <div class="alert alert-warning">
+    <b>Important:</b> Do not navigate away from this page while saving the
+    certificate and key files. They are unique and won't display again.</div>
+
+5. Store the files on your runtime's local filesystem.
+
+Next, configure a {{site.base_gateway}} runtime using the
+certificate, the private key, and the remaining configuration details on the
+**Configure Runtime** page.
+
+{% navtabs %}
+{% navtab Using Docker %}
+
+1. (Optional, if no gateway data plane exists) Using the
+[Docker installation documentation](/enterprise/latest/deployment/installation/docker),
+follow the instructions to:
+    1. [Download {{site.base_gateway}} 2.3.0.0 or later](/enterprise/latest/deployment/installation/docker#pull-image).
+    2. [Create a Docker network](/enterprise/latest/deployment/installation/docker/#create-network).
+
+    <div class="alert alert-warning">
+        Do not start or create a database on this node.
+    </div>
+
+2. In the **Configuration Parameters** section, copy the codeblock.
+
+3. Replace the values in `KONG_CLUSTER_CERT`, `KONG_CLUSTER_CERT_KEY`,
+and `KONG_CLUSTER_CA_CERT` with the paths to your certificate files.
+
+4. Apply the parameters to your gateway instance as environment variables and
+reload {{site.base_gateway}}.
+
+    See [Parameters](#parameters) for descriptions and the matching fields in
+    {{site.konnect_short_name}}.
+
+    ```sh
+    $ echo "KONG_ROLE=data_plane \
+      KONG_DATABASE=off \
+      KONG_ANONYMOUS_REPORTS=off \
+      KONG_CLUSTER_MTLS=pki \
+      KONG_CLUSTER_CONTROL_PLANE=<example.cp.konnect.foo>:443 \
+      KONG_CLUSTER_SERVER_NAME=<kong-cpoutlet-example.service> \
+      KONG_CLUSTER_TELEMETRY_ENDPOINT=<example.tp.konnect.foo>:443 \
+      KONG_CLUSTER_TELEMETRY_SERVER_NAME=<kong-telemetry-example.service> \
+      KONG_CLUSTER_CERT=/<path-to-file>/cluster.crt \
+      KONG_CLUSTER_CERT_KEY=/<path-to-file>/cluster.key \
+      KONG_LUA_SSL_TRUSTED_CERTIFICATE=/<path-to-file>/ca.crt \
```

Comment:
@artomal In the hybrid mode instructions, we also include `KONG_CLUSTER_CA_CERT`. According to the [Admin API doc](https://docs.konghq.com/enterprise/2.2.x/property-reference/#cluster_ca_cert), this is required when `cluster_mtls` is set to `pki`, but I didn't see it in the eng PR and previews. Is it no longer required?

Reply:
I think if you include it in `cluster_cert` you don't need to set `cluster_ca_cert` correct me if I am wrong @fffonion

Reply:
If this is for bring-your-own-data-plane then `cluster_ca_cert` is still needed, `cluster_cert` is the client cert DP present to CP, and `cluster_ca_cert` is the CA for DP to verify CP.

Also we should use `KONG_LUA_SSL_TRUSTED_CERTIFICATE=system,/<path-to-file>/ca.crt` for including OS default trusted CA as well.

Reply:
If this is for bring-your-own-data-plane then `cluster_ca_cert` is not needed, `cluster_cert` is the client cert DP present to CP, and `cluster_ca_cert` is the CA for CP to verify DP.

Also we should use `KONG_LUA_SSL_TRUSTED_CERTIFICATE=system,/<path-to-file>/ca.crt` for including OS default trusted CA as well.

Reply:
Is `system` just that, ie a user would literally include `system` in the value? Not a variable?

Reply:
@fffonion I think you added system by mistake no? I usually add just a path like in the example like this `KONG_LUA_SSL_TRUSTED_CERTIFICATE=/<path-to-file>/ca.crt` and it works just fine at least for BYODP case.

Reply:
yes `system` is just literally the value. it's the default value starting  2.2 (https://docs.konghq.com/gateway-oss/2.2.x/configuration/#lua_ssl_trusted_certificate). adding `system` does not influence hybrid mode, but some use case that requires lua land ssl verify will need the `system` to be set (acme plugin for example). so it's always a good pratice to append new path after `system`.